WcaOnRails/db/migrate/20190803202212_update_wca_states.rb

```diff
@@ -0,0 +1,21 @@
+class UpdateWcaStates < ActiveRecord::Migration[5.2]
+  def up
+    Country.delete_all
+    Country::ALL_STATES.each(&:save!)
+    # Extra changes due to some changes in country names
+    models=[Person, Result, Competition]
+    models.each do |m|
+      m.where(countryId: "Macedonia").update(countryId: "North Macedonia");
```

Comment:
I think we want `update!` instead of `update` here, so we find out if any updates fail.

Reply:
since this is an `ActiveRecord`'s relation there is no `update!` method there!
But actually I changed it to `update_all`, because `update` triggers the object validation and callbacks whereas `update_all` just fires the updates.
(and I believe this is what we want, since we just want to substitute an existing country name for the new one)

Reply:
Ah, I agree that `update_all` makes the most sense here.